cirq-core/cirq/ops/dense_pauli_string.py

```diff
@@ -614,10 +614,12 @@ def _as_pauli_mask(val: Iterable[cirq.PAULI_GATE_LIKE] | np.ndarray) -> np.ndarr
 
 
 def _attempt_value_to_pauli_index(v: cirq.Operation) -> tuple[int, int] | None:
-    if not isinstance(v, raw_types.Operation):
+    if isinstance(v, pauli_string.PauliString):
```

Comment:
Maybe this should go into `_try_interpret_as_pauli_string` itself?

Reply:
Or is this what you meant you already tried in your earlier comment?

Reply:
Please ignore the above question... I figured out that after relaxing the type check in _try_interpret_as_pauli_string to "Operation", this problem resolved automatically, no need to update `_mul_with_qubits`.